common/changes/@itwin/manage-versions-react/vp-Update-mange-versions-package_2025-05-05-13-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/manage-versions-react",
+      "comment": "Update Manage-Version to itwin-react V3",
+      "type": "minor"
```

Comment:
@veekeys 
Shouldn't this be a major bump?

Reply:
well, there is no breaking change here.. So why?

Reply:
going from itwinui@2 -> itwinui@3 is breaking

Reply:
What exactly is breaking to consumer of manage versions component?

Reply:
parent app needs to now import the stylesheets not the pkg
take a look at the other pkgs in this monorepo that we had to recently update to itwinui v3

Reply:
recommended, but works without it too as iui handles it if styles are not present.

Reply:
okl resolving, merging and publishing. if there are issue, please create new pr to make this a mjaor bump